src/components/LogRow/ResmokeRow/ResmokeRow.test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { LogContextProvider } from "context/LogContext";
+import { renderWithRouterMatch, screen, userEvent, waitFor } from "test_utils";
+import { ResmokeRow } from ".";
+
+const wrapper = (logs: string[]) =>
+  // eslint-disable-next-line func-names, react/display-name
+  function ({ children }: { children: React.ReactNode }) {
+    return (
+      <LogContextProvider initialLogLines={logs}>{children}</LogContextProvider>
+    );
+  };
```

Comment:
Alternatively maybe like this?

```suggestion
const wrapper = (logs: string[]) => {
  const provider = ({ children }: { children: React.ReactNode }) => (
    <LogContextProvider initialLogLines={logs}>{children}</LogContextProvider>
  );
  return provider;
}
```